src/pydata_sphinx_theme/assets/scripts/index.js

```diff
@@ -9,6 +9,10 @@ import "bootstrap";
 
 import "../styles/index.scss";
 
+////////////////////////////////////////////////////////////////////////////////
+// TOC interactivity
+////////////////////////////////////////////////////////////////////////////////
+
```

Comment:
```suggestion
/**
 * TOC interactivity
 */

```

Let's use the typescript comment block style here: https://marketplace.visualstudio.com/items?itemName=salbert.comment-ts